dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -45,22 +57,18 @@ cmd_config() {
         log "rm -rf \"${CHE_HOST_INSTANCE}/dev\" >> \"${LOGS}\""
         rm -rf "${CHE_CONTAINER_INSTANCE}/dev"
     fi
+
+    if [[ ! -d $(echo ${CHE_CONTAINER_DEVELOPMENT_REPO}/${CHE_ASSEMBLY_IN_REPO}) ]]; then
```

Comment:
I would like us to move this into the docker.sh, so that all of the checks of the pre-reqs (in this case modules properly built) is handled before normal execution of a command.

In this method, we should then take actions that assume the presence of the pre-reqs.